tokio-current-thread/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "tokio-current-thread"
+
+# When releasing to crates.io:
+# - Update html_root_url.
+# - Update CHANGELOG.md.
+# - Create "v0.1.x" git tag.
+version = "0.1.6"
```

Comment:
This should be "0.1.0" since it is the first release.

Reply:
I've changed the crate to the 0.1.0 version and I've also adjusted deprecated annotations which were pointing on the non existing 0.1.2 version.